README.md

```diff
@@ -53,6 +53,14 @@ runs out.
 A queue of inbound attachments that are pending download. If downloading fails, downloads will be
 re-queued here for a limited period of time until they either send successfully or time runs out.
 
+### /intro/from/fromIdentityKey/toIdentityKey
```

Comment:
@Kallirroi This is the path you'll want to listen to for building the Introductions list. It's already ordered by from (who sent the introduction) so you don't have to sort it yourself. Remember, this is just an index pointing to the actual StoredMessage, so you'll have to subscribe for details (just like we do on contact messages for the conversation).